### DISCUSSION TITLE
Externalizable support

### DISCUSSION THREAD

**Discussion by exabrial** [eclipse-serializer/serializer#153] [Q&A]
I'm trying to serialize an OpenWebBeans context. It implements Externalizable. JDK Serialization seems to pick up on this automatically. I can't figure out how to configure serializer to do this automatically. 

Would greatly appreciate if someone could point me in the correct direction. Thank you!

**Comment by hg-ms** [comment]
We do not use the java interfaces like Serializable and Externalizable. For serialization we rely on reflections to analyze and access objects and their data. This is either done by specialized type handler implementations or a dynamic class analysis and handler creation at runtime when a class is going to be serialized the first time.
If those are not sufficient there is also the option to provide custom handler implementation for specific classes.

Please see the [documentation](https://docs.eclipsestore.io/manual/serializer/index.html) for the first steps.
Please be aware that there a different options how meta information of serialized classes is handles.
